openlattice.yaml

```diff
@@ -1442,6 +1442,95 @@ paths:
               schema:
                 type: integer
             
+  ###################
+  ## DATA SET APIs ##
+  ###################
+  
+  /datastore/organization-database/{organizationId}/external-database-table:
+    get:
+      summary: Gets all OrganizationExternalDatabaseTable objects for an organization
+      operationId: getExternalDatabaseTables
+      tags: [dataset]
+      security:
+        - openlattice_auth: []
+        - http_auth: []
+      parameters:
+        - name: organizationId
+          in: path
+          required: true
+          schema:
+            type: string
+            format: uuid
+      responses:
+        "200":
+          description: Success
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/OrganizationExternalDatabaseTable"
+  /datastore/organization-database/{organizationId}/external-database-table/external-database-column:
+    get:
+      summary: Gets a map of all OrganizationExternalDatabaseTable objects to OrganizationExternalDatabase columns that are contained within each table.
+      operationId: getExternalDatabaseTablesWithColumns
+      tags: [dataset]
+      security:
+        - openlattice_auth: []
+        - http_auth: []
+      parameters:
+        - name: organizationId
+          in: path
+          required: true
+          schema:
+            type: string
+            format: uuid
+      responses:
+        "200":
+          description: Success
+          content:
+            application/json:
+              schema:
+                type: object
+                additionalProperties:
+                  type: array
+                  items:
+                    $ref: "#/components/OrganizationExternalDatabaseColumn"
+  /datastore/organization-database/{organizationId}/{tableId}/external-database-table/external-database-column:
+    get:
+      summary: Gets an object containing an OrganizationExternalDatabaseTable object and its OrganizationExternalDatabase columns for an organization
+      operationId: getExternalDatabaseTableWithColumns
+      tags: [dataset]
+      security:
+        - openlattice_auth: []
+        - http_auth: []
+      parameters:
+        - name: organizationId
+          in: path
+          required: true
+          schema:
+            type: string
+            format: uuid
+        - name: tableId
+          in: path
+          required: true
+          schema:
+            type: string
+            format: uuid
+      responses:
+        "200":
+          description: Success
+          content:
+            application/json:
+              schema:
+                type: object
+                
+                additionalProperties:
+                  type: array
```

Comment:
This is only returning 1 `OrganizationExternalDatabaseTableColumnsPair` and not an array.

Reply:
done